code-generators/npm-schema-only-package/src/package-template.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
In the top level `package.json` there is a `scripts` section `copy-pkg-misc` which copies the `package-template.json` and `README.md` to `code-generators/typescript-types/dist`. It would be worth creating two new script sections one to copy the `typescript-types` files (e.g. `copy-pkg-typescript-types`) and another to copy the `npm-schema-only` package (e.g. `copy-npm-schema-only`), the update the `copy-pkg-misc` to run them both e.g. `copy-pkg-typescript-types && copy-npm-schema-only`.